workspace/popjava/src/popjava/dataswaper/POPByte.java

```diff
@@ -0,0 +1,76 @@
+package popjava.dataswaper;
+
+import popjava.buffer.POPBuffer;
+/**
+ * Primitive settable byte for POPJava, needed for {@link popjava.annotation.POPParameter} direction use in methods 
```

Comment:
From [POPConnectorJobManager.java#L81-L82](https://github.com/MDosky/pop-java/blob/job-manager/workspace/popjava/src/popjava/service/jobmanager/protocol/POPConnectorJobManager.java#L81-L82)
To [POPJavaJobManager.java#L529](https://github.com/MDosky/pop-java/blob/job-manager/workspace/popjava/src/popjava/service/jobmanager/POPJavaJobManager.java#L529)